style.css

```
@@ -1,7 +1,159 @@
+/* Root Styles from Style-Guide.md */
+:root {
+  --pale-blue: hsl(225, 100%, 94%);
+  --bright-blue: hsl(245, 75%, 52%);
+
+  --very-pale-blue: hsl(225, 100%, 98%);
+  --desaturated-blue: hsl(224, 23%, 55%);
+  --dark-blue: hsl(223, 47%, 23%);
+
+  /* Typography and Spacing */
+  --font-family: "Red Hat Display", sans-serif;
+  --font-size-body: 16px;
+
+  --padding: 1.5rem;
+  --border-radius: 12px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-height: 100vh;
+  padding-top: 5rem;
+  font-family: var(--font-family);
+  font-size: var(--font-size-body);
+  background: url(./images/pattern-background-mobile.svg) top center
+    no-repeat;
+  background-color: var(--pale-blue);
+  margin: 0;
+  overflow-y: auto;
+}
+
+.container {
+  background: #fff;
+  border-radius: var(--border-radius);
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.1);
+  text-align: center;
+  max-width: 375px;
+  width: 100%;
+  overflow: hidden;
+  font-weight: 500px;
+}
+
+.image-container {
+  width: 100%;
+}
+
+.content {
+  padding: 24px 40px;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.header {
+  color: var(--dark-blue);
+  font-weight: 900;
+  font-size: 22px;
+}
+
+p {
+  color: var(--desaturated-blue);
+  font-size: 16px;
+}
```

```suggestion
p {
  color: var(--desaturated-blue);
  font-size: 16px;
  padding-inline: 1rem;
}
```
I added a left and right padding of 1rem to get the body text to look like the design